### DISCUSSION TITLE
Amd

### DISCUSSION THREAD

**Discussion by elen07zz** [vladmandic/sdnext#45] [General]
Can this be used with an amd gpu?

**Comment by vladmandic** [comment]
yes, although i don't test it. and sorry for the late reply, for some reason i wasn't getting notifications on the new discussion topics.

**Reply by eneskuluk** [reply]
Can it be used on windows or AMD is exclusive for Linux? There is directml fork of automatic that works on windows for AMD GPUs. Maybe we can import directml into this fork or that fork could maybe based on this fork

**Comment by vladmandic** [comment]
yes, i've started investigating, but cannot promise when i'll make progress on that.

**Reply by motorist828** [reply]
Is there any progress on this?

**Reply by vladmandic** [reply]
monitor issue thread #217

**Comment by motorist828** [comment]
Directml version of this fork appeared
i liked your work, also thank lshqqytiger
I have an increase in speed of 16-18%. (vega 56)
I want to completely switch to your version when it becomes more stable
 https://github.com/lshqqytiger/stable-diffusion-webui-directml/discussions/74#discussioncomment-5698790

**Reply by vladmandic** [reply]
nice! i just reached out to @lshqqytiger if he's interested in contributing directly!
